File: SchoolAppClient.NG/src/app/Models/exam-schedule-vm.ts
import { ExamScheduleStandardForExamScheduleVm } from "./exam-schedule-standard-for-exam-schedule-vm";

export class ExamScheduleVm {
  examScheduleId !: number;
  examScheduleName !: string;
  examScheduleStandards !: ExamScheduleStandardForExamScheduleVm[];
}

File: SchoolApiService/ViewModels/ExamScheduleVM.cs
namespace SchoolApiService.ViewModels
{
    public class ExamScheduleVM
    {
        public int ExamScheduleId { get; set; }
        public string? ExamScheduleName { get; set; }
        public IEnumerable<ExamScheduleStandardForExamScheduleVM>? ExamScheduleStandards { get; set; } = [];
    }
}
